tests/acceptance/tests/Account/ForgotPassword.spec.ts

```diff
@@ -9,17 +9,17 @@ test ('As a customer, I can request a new password with existing customer email
     const customer = await TestDataService.createCustomer();
     await test.step('Navigate to the login page and click on forgot password', async () => {
         await ShopCustomer.goesTo(StorefrontAccountLogin.url());
-        await StorefrontAccountLogin.forgotPasswordLink.click();
+        await ShopCustomer.presses(StorefrontAccountLogin.forgotPasswordLink);
     });
 
     await test.step('Fill in the customer email and request a password reset', async () => {
         await StorefrontAccountRecover.emailInput.fill(customer.email);
```

Comment:
```suggestion
        await ShopCustomer.fillsIn(StorefrontAccountRecover.emailInput, customer.email);
```
marking the lines where the .fillsIn() method is missing :)